lib/colorls/flags.rb

```diff
@@ -23,38 +23,39 @@ def initialize(*args)
 
     def process
       incompatible = report_incompatible_flags
```

Comment:
Remove variable `incompatible` and rename method `report_incompatible_flags` to `incompatible_flags?`, to be used in this method.